Resources/translations/CmfMenuBundle.cs.xliff

```diff
@@ -80,7 +80,7 @@
       </trans-unit>
       <trans-unit id="form.label_parent">
         <source>form.label_parent</source>
-        <target>Rodič</target>
+        <target>Menu</target>
```

Comment:
I've renamed from "parent" to "Menu" becuase I think it makes more sense. As "Menu" was already used a couple of times in the message catalogues, I was able to update all languages.